src/urbit/lib/chess.hoon

```diff
@@ -1407,7 +1409,8 @@
 ::  a test game
 ::  a game that ends in four moves with black side winning
 ++  fools-mate
-  ^-  chess-game
+::  XX: bad hack to get this going, after changing chess-game info
+::  ^-  chess-game
```

Comment:
This isn't necessary - the distinction of whether a game is active or archived is made by how it's stored. We only surface that information to the frontend with a `'archived'` field in a JSON object.

Reply:
this comment was leftover from the time `chess-game` was being tweaked to handle archived games too. it broke this part of the code. but that's all reverted back now.